src/main/components/email/Integrations.jsx

```diff
@@ -2,21 +2,23 @@ import React from "react";
 import Content from "../common/content/Content";
 import CardComponent from "../../molecules/card/Card";
 
-const IntegrationComponent = () => {
+import { integrateGmail } from '../../../utils/integrationUtils';
+
+const IntegrationComponent = (props) => {
   const listOfIntegrations = [
```

Comment:
maybe a better name would be listOfAvaliableEmails?

Reply:
Maybe list of listOfAvalaibleIntegrations